prow/cmd/deck/static/common/common.ts

```diff
@@ -216,65 +215,19 @@ export function getCookieByName(name: string): string {
   return "";
 }
 
-export function createRerunProwJobIcon(modal: HTMLElement, rerunElement: HTMLElement, prowjob: string, rerunCreatesJob: boolean, csrfToken: string): HTMLElement {
-  const url = `${location.protocol}//${location.host}/rerun?prowjob=${prowjob}`;
-  const i = icon.create("refresh", "Show instructions for rerunning this job");
-
-  window.onkeydown = (event: any) => {
-    if ( event.key === "Escape" ) {
-      modal.style.display = "none";
-    }
-  };
-  window.onclick = (event: any) => {
-    if (event.target === modal) {
-      modal.style.display = "none";
-    }
-  };
-
-  // we actually want to know whether the "access-token-session" cookie exists, but we can't always
-  // access it from the frontend. "github_login" should be set whenever "access-token-session" is
-  i.onclick = () => {
-    modal.style.display = "block";
-    rerunElement.innerHTML = `kubectl create -f "<a href="${url}">${url}</a>"`;
-    const copyButton = document.createElement('a');
-    copyButton.className = "mdl-button mdl-js-button mdl-button--icon";
-    copyButton.onclick = () => copyToClipboardWithToast(`kubectl create -f "${url}"`);
-    copyButton.innerHTML = "<i class='material-icons state triggered' style='color: gray'>file_copy</i>";
-    rerunElement.appendChild(copyButton);
-    if (rerunCreatesJob) {
```

Comment:
I honestly don't know what "rerunCreatesJob" means. I understand it's an input parameter. It's unfortunately named after what it's _doing_, not what it _means_. 😢  I think we should rename it -- maybe like "userHasPermissionToRerun" or "allowRerun" or similar??

.... This was original code, but I'll recommend it for the code moved to `rerun.ts`. (Same with the below.)

Reply:
It based on current functionality, it is an option that is set that determines if a job could create a job at rerun. This variable is being passed by Go into the UI. I would rename it, but the Go code uses the same naming convention, and I'd prefer to keep the same consistency here.

Reply:
Yeah. Thanks, golang. 😐 🤣